frontend/src/types.ts

```diff
@@ -669,7 +669,7 @@ export type gpuScale = {
 export type GPUInfo = {
   configured: boolean;
   available: number;
-  autoscalers: gpuScale[];
+  autoscalers?: gpuScale[];
```

Comment:
According to the backend, this is always an array.
```suggestion
  autoscalers: gpuScale[];
```